src/utils/project_folder.rs

```diff
@@ -0,0 +1,46 @@
+use std::{env, path::PathBuf};
```

Comment:
Can you rename this module to `project_directory`?

Reply:
Renamed